meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -1,183 +1,213 @@
 inherit image_types
 
-# Images inheriting this class MUST define:
+#
+# Create a raw image that can by written onto a storage device using dd/etcher.
+#
 # RESIN_IMAGE_BOOTLOADER        - bootloader
-# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed on boot partition
+# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed
+#                               - on boot partition
 #                               - items need to be in the 'src:dst' format
 #                               - src needs to be relative to DEPLOY_DIR_IMAGE
 #                               - dst needs to be an absolute path
-#                               - if dst is ommited ('src:' format used), absolute path of src will be used as dst
-#
-# Optional:
-# RESIN_SDIMG_ROOTFS_TYPE       - rootfs image to be used [default: ext4]
-# RESIN_BOOT_SPACE              - size of boot partition in KiB [default: 40960]
-# RESIN_SDIMG_COMPRESSION       - define this to compress the final SD image with gzip, xz or bzip2 [default: empty]
-
-#
-# Create an image that can by written onto a SD card using dd.
+#                               - if dst is ommited ('src:' format used),
+#                               - absolute path of src will be used as dst
+# RESIN_ROOT_FSTYPE             - rootfs image type to be used for integrating
+#                               - in the final raw image
+# RESIN_BOOT_SIZE               - size of boot partition in KiB
+# RESIN_RAW_IMG_COMPRESSION     - define this to compress the final war image
+#                               - with gzip, xz or bzip2
 #
-# The disk layout used is:
+# Partition table:
 #
-#    0                      -> IMAGE_ROOTFS_ALIGNMENT         - reserved for other data
-#    IMAGE_ROOTFS_ALIGNMENT -> RESIN_BOOT_SPACE               - boot partition (usually kernel and bootloaders)
-#    RESIN_BOOT_SPACE       -> ROOTFS_SIZE                    - rootfs
-#    ROOTFS_SIZE            -> UPDATE_SIZE                    - update partition (this is a duplicate of rootfs so UPDATE_SIZE == ROOTFS_SIZE)
-#    UPDATE_SIZE            -> SDIMG_SIZE                     - extended partition
+#   +-------------------+
+#   |                   |  ^
+#   | Reserved          |  |RESIN_IMAGE_ALIGNMENT
+#   |                   |  v
```

Comment:
@floion we discussed in the arch call and we decided to stay on 700M as entire space taken by resin. So we will have from now on the rootfs of 308MiB as stated in this PR.